.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code
+# and run tests across different versions of node
+# For more information see:
+# https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
We should probably move to v12 and forget about v10.

However we would need to migrate to a v12 base-image for that.

Reply:
Switched to use `node12` only. The recent changes (rebased) introduce a node 12 container, so we should be good...

Reply:
It's nice to at least unit test on 14 as well. They are quick to run and they might flag up future problems.

Reply:
But that can be sorted in another PR.

Reply:
Fixed